doc/src/base/arrays.md

```diff
@@ -128,6 +128,7 @@ Base.reinterpret
 Base.reshape
 Base.dropdims
 Base.vec
+Base.SubArray
```

Comment:
So the build failure was detecting a genuine issue

> ```
> Warning: no doc found for reference '[`SubArray`](@ref)' in src/base/arrays.md.
> ```
>
> --- https://build.julialang.org/#/builders/20/builds/6554

I suggest to just include it in the documentation. @mbauman What do you think?

FYI the docstring ATM is:

https://github.com/JuliaLang/julia/blob/377aa809eba31edbf9d73d22b85e801cd4664151/base/subarray.jl#L8-L12